### ISSUE TITLE
Split "status" command into "status" and "stats"

### ISSUE THREAD

**Issue by zuzak** [grifferz/pah-irc#78]
Currently, `status` lists whether a game is active, the current Card Tsar, the active players, the leaderboard, and the current black card. It would be nice to split this into two separate commands: `status` for information pertinent to the current game (current card, Tsar, who we're waiting on, maybe active players), and `stats` for information about the "meta" game (leaderboard, maybe last winning combination). Six (beepy) lines is a tad much for a single command.

**Comment by grifferz** [comment]
Right then. `status` currently looks like this:
- `<AgainstHumanity>` A game is active in **#bitfolk**! We're just waiting on **dotwaffle** to make their play.
- `<AgainstHumanity>` The current Card Tsar in **#bitfolk** is **ging**
- `<AgainstHumanity>` Active Players in **#bitfolk**: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1) tommy(0) dutchie(0) plett(0) daviey(0) dg(0) dotwaffle(0) dee(0)
- `<AgainstHumanity>` Top 3 all time in **#bitfolk**: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1)
- `<AgainstHumanity>` Current Black Card in **#bitfolk**:
- `<AgainstHumanity>` → **What are school administrators using to curb rampant teenage pregnancy?**

Issue #66 proposes to change it to:
- `<AgainstHumanity>` Waiting on **dotwaffle** to make their play. Round started about **18 hrs, 20 mins ago**. Idle punishment in about **22 hrs, 50 mins**.
- `<AgainstHumanity>` The current Card Tsar in **#bitfolk** is **ging**.
- `<AgainstHumanity>` Active Players in **#bitfolk**: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1) tommy(0) dutchie(0) plett(0) daviey(0) dg(0) dotwaffle(0) dee(0)
- `<AgainstHumanity>` Top 3 all time in **#bitfolk**: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1)
- `<AgainstHumanity>` Current Black Card in **#bitfolk**:
- `<AgainstHumanity>` → **What are school administrators using to curb rampant teenage pregnancy?**

With this issue I propose that `status` give us:
- `<AgainstHumanity>` [**#bitfolk**] Waiting on **dotwaffle** to make their play. Round started about **18 hrs, 20 mins ago**. Idle punishment in about **22 hrs, 50 mins**.
- `<AgainstHumanity>` [**#bitfolk**] Active Players: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1) tommy dutchie plett daviey dg dotwaffle dee
- `<AgainstHumanity>` [**#bitfolk**] The Card Tsar is **ging**; current Black Card:
- `<AgainstHumanity>` → **What are school administrators using to curb rampant teenage pregnancy?**

That's for private message; no [**#bitfolk**] bit for channel message.

And that `stats` give us:
- `<AgainstHumanity>` [**#bitfolk**] Active Players: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1) tommy dutchie plett daviey dg dotwaffle dee
- `<AgainstHumanity>` [**#bitfolk**] Top 3 all time in **#bitfolk**: grifferz(2) zuzak(2) dominic(1) ziphy(1) popey(1) tolien(1) dave2(1) filbert(1) ging(1) johnm(1)

plus more we think of later…

**Comment by chrisjrob** [comment]
I would suggest removing ActivePlayers as well from the status command. That is not required information to play.

**Comment by grifferz** [comment]
Okay. But I'm not sure that fits into the spirit of `stats` either!

**Comment by chrisjrob** [comment]
That is true! But the status command is one that you use so often, it needs to be as short as possible and not have any information that is not required to play.
Perhaps move ActivePlayers to a "players" or "names" command.
